deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -88,300 +106,297 @@ <h4 class="panel-title pull-left">Capacity</h4>
-                    value: info.defaultCMPConfigs[key]
-                }
-            }).filter(function(item){return item.name!='cmp_group'}),
-        assignPublicIP: false,
-        baseImageHelpData: [],
-        baseImages: mapBaseImagesToOptions(baseImagesSorted),
-        baseImageValue: defaultImageId,
-        showPinImage: enable_ami_auto_update,
-        pinImage: enable_ami_auto_update ? goldenImage == null : true,
-        pinImageEnabled: isPinImageEnabled(info.defaultBaseImage),
-        configOptions: Object.keys(info.configList).map(
-            function (key) {
-                return {
-                    name: key,
-                    text: key,
-                    default: info.configList[key]
-                }
-            }).filter(function(item){return item.name != 'assign_public_ip'}),
-        confirmDialogTitle: "Confirm New Capacity Creation",
-        confirmDialogId: "createHostGroup",
-        currentProvider: info.defaultProvider,
-        currentCell: info.defaultCell,
-        currentArch: info.defaultArch,
-        hostTypeHelpData:[],
-        hostTypeOptions: info.hostTypes.map(
-            function (item, idx) {
-                return {
-                    value: item.id,
-                    text: item.abstract_name + " (" + item.core + " cores, " + item.mem + " GB, " + item.storage + ", " + item.network + ", " + item.provider + ": " + item.provider_name + ", " + item.network + ")",
-                    isSelected: item.abstract_name === info.defaultHostType,
-                    isDisabled: item.retired
-                }
-            }),
-        cellValue: info.defaultCell,
-        archValue: info.defaultArch,
-        imageNameValue: info.defaultBaseImage,
-        imageNames: mapImageNameToOptions(info.baseImageNames),
-        instanceCount: "",
-        placements: placements.getFullList(false, null),
-        placementsHelpData: [],
-        providers: info.providerList.map(
-            function (o) {
-                return {
-                    value: o, text: o
+    var capacitySetting = new Vue({
+        el: "#mainPanel",
+        data: {
+            allUserData: Object.keys(info.defaultCMPConfigs).map(
+                function (key) {
+                    if (key == 'access_role') {
+                        info.defaultCMPConfigs[key] = localStorage.getItem("accessRole") ? localStorage.getItem("accessRole") : info.defaultCMPConfigs[key];
+                    }
+                    return {
+                        name: key,
+                        value: info.defaultCMPConfigs[key]
+                    }
+                }).filter(function (item) { return item.name != 'cmp_group' }),
+            assignPublicIP: false,
+            baseImageHelpData: [],
+            baseImages: mapBaseImagesToOptions(baseImagesSorted),
+            baseImageValue: defaultImageId,
+            pinImage: getPinImageValue(goldenImage, true), // Default auto update if there is a golden image 
```

Comment:
We should be careful not to publish Golden image util it's GA.

Reply:
Good point. Default to Pin Image until GA will be better.